fastlane/Fastfile

```diff
@@ -175,6 +175,7 @@ end
   lane :new_beta_release do | options |
     ios_betabuild_prechecks(options)
     ios_update_metadata(options)
+    sanitize_appstore_keywords()
```

Comment:
Just FYI, I learned from RuboCop that we can omit the `()` in methods that take no arguments in Ruby.

```suggestion
    sanitize_appstore_keywords
```